plugin/src/index.js

```diff
@@ -24,3 +24,4 @@ export default presetWarp
 export { theme } from '#theme'
 export { rules } from '#rules'
 export { variants } from '#variants'
+export * as bounds from '#bounds'
```

Comment:
Why do we need to export bounds here, too?

Reply:
So the docs (and maybe other places) can consume it